app/containers/LoginNep2/LoginNep2.jsx

```diff
@@ -39,7 +39,7 @@ export default class LoginNep2 extends Component<Props, State> {
             />
           </div>
           <PasswordInput
-            placeholder="Enter your passphrase here"
+            placeholder="Password"
```

Comment:
Up on line 34 is a verbose placeholder "Enter your encrypted key here".  Should we update that as well, perhaps to say "Encrypted Key"?

Reply:
👍 sounds good @mhuggins